shared/django_apps/codecov/models.py

```diff
@@ -11,3 +11,12 @@ class BaseCodecovModel(models.Model):
 
     class Meta:
         abstract = True
+
+
+class BaseModel(models.Model):
```

Comment:
Any reason to not use the `BaseCodecovModel`? (I don't have an opinion on using it... just wanted to know your reasoning).

Reply:
the only difference is that `BaseCodecovModel` adds an `external_id` field. I don't understand why that's part of our base model - it's a non-editable uuid field. Since I couldn't think of a reason for it, it seemed like unnecessary bloat, so I left it off the new tables.
I'll raise it with the team to see if there's any context